### PR TITLE
Fix form tooltip positioning

### PR DESCRIPTION
**Pull request by weeman1337**
Closes https://github.com/vector-im/element-web/issues/22861

Before

tooltip_before

After

tooltip_after

The screen from the linked issue after

tooltip_after2

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix form tooltip positioning ([\#9598](https://github.com/matrix-org/matrix-react-sdk/pull/9598)). Fixes vector-im/element-web#22861.

### CODE REVIEW

File: res/css/views/elements/_Field.pcss [hunk 1]
```diff
@@ -174,8 +174,6 @@ limitations under the License.
 }
 
 .mx_Field_tooltip {
-    margin-top: -12px;
-    margin-left: 4px;
     width: 200px;
 }
```
Comment (richvdh) [inline res/css/views/elements/_Field.pcss:178]: Afraid I'm not familiar with this stuff. Where (else) is `.mx_Field_tooltip` used, why was this necessary in the first place, and why is it no longer necessary?

Comment (weeman1337) [reply res/css/views/elements/_Field.pcss:178]: Good questions!

`.mx_Field_tooltip` is only used for `Field`s. Means it should not break other places™

I don't know the exact reason for the margin. But it seems like they are not so great aligned [when they were introduced](https://github.com/matrix-org/matrix-react-sdk/pull/2550).

image

**Review by richvdh — APPROVED** [review]
seems plausible
